src/bridge/components/Header.js

```diff
@@ -31,28 +32,30 @@ const Crumbs = props => {
   });
 
   return rendered;
-};
-
-const Header = props => {
-  const crumbs = (
-    <div className={'flex items-center h-10'}>
-      <Crumbs
-        routeCrumbs={props.routeCrumbs}
-        skipRoute={props.skipRoute}
-        networkType={props.networkType}
-        wallet={props.wallet}
-        pointCursor={props.pointCursor}
-      />
-    </div>
-  );
+}
+
+function Header(props) {
+  const history = useHistory();
+
+  const showCrumbs = !history.includes(ROUTE_NAMES.INVITE_TICKET);
```

Comment:
@Fang- which inner view do mean?

Reply:
ie specify "should this page display breadcrumbs or not" on the view/component as found in `/src/bridge/views` (the different "pages" we route to), rather than having a check like this in `Header`.

Reply:
ah, yeah, that's totally something we should decide on, pattern-wise. are we making a distinction between "dumb" components that just interact with their props and isolated state and "connected" components that know about higher-level state and use our HOCs/Hooks for connecting data sources to dumb components. In our case, views would be connected components and ideally all of the indigo components and re-usable bridge-specific things are "dumb".

If that's a distinction we want to maintain, I can make it a priority to separate the concerns in the next few pull requests.

Personally in recent projects I've ended up giving up on dumb vs connected component distinctions (except for isolated libraries like our indigo-react) just because it was a pain to handle all of the boiler plate and maintain multiple component definitions & files per component.

here's dan's thoughts on this concept from 2015 for context https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0